### PR TITLE
chore: bump PHP microgen to v0.1.2 for REGAPIC support

### PR DESCRIPTION
**Pull request by miraleung**

### CODE REVIEW

File: google/cloud/compute/v1/compute_gapic.yaml [hunk 1]
```diff
@@ -18,15 +18,4 @@ config_schema_version: 2.0.0
...
     package_name: com.google.cloud.compute.v1
-  python:
-    package_name: google.cloud.compute_v1.gapic
-  go:
-    package_name: cloud.google.com/go/compute/apiv1
-  csharp:
-    package_name: Google.Compute.V1
-  ruby:
-    package_name: Google::Cloud::Compute::V1
-  php:
```
Comment (bshaffer) [inline google/cloud/compute/v1/compute_gapic.yaml:29]: Are these ok to remove? Just making sure this is intentional

**Review by bshaffer — APPROVED** [review]

File: google/cloud/compute/v1/compute_gapic.yaml (hunk 1 above)
Comment (miraleung) [reply google/cloud/compute/v1/compute_gapic.yaml:29]: Yes - only the Java and PHP microgenerators will ever use the `long_running` and `batching` options in this file now. This file can be deleted once Java migrates as well.

File: WORKSPACE [hunk 2]
```diff
@@ -353,7 +353,7 @@ php(
 php_gapic_repositories()
 
 # PHP micro-generator (beta)
-_gapic_generator_php_version = "0.1.1"
+_gapic_generator_php_version = "0.1.2"
```
Comment (vam-google) [inline WORKSPACE:356]: It is Ok for this PR, but please also update it in googleapis to keep the two workspace files in sync

**Review by vam-google — APPROVED** [review]
LGTM

**Pull request merged**